App.tsx

```diff
@@ -13,9 +17,16 @@ const App = () => {
 
 	return (
 		<GestureHandlerRootView style={{ flex: 1 }}>
-			<Providers>
-				<Routes />
-			</Providers>
+			<WalletConnectProvider
```

Comment:
@romulloqueiroz I could not make it work properly with the providers we have right now so I used it outside the providers list.